manifests/init.pp

```diff
@@ -89,6 +90,7 @@
     create_resources('firewalld_zone',      $zones)
     create_resources('firewalld_service',   $services)
     create_resources('firewalld_rich_rule', $rich_rules)
+    create_resources('firewalld::custom_service', hiera('firewalld::custom_service', $custom_services))
```

Comment:
You're right.. I tried a few things yesterday, but it was not working, then I left it that way. I'm not really a Puppet developer :) Thanks